src/stories/components/delegate-summary/delegate-summary.stories.tsx

```diff
@@ -47,41 +58,41 @@ LightMode.parameters = {
     component: {
       375: {
         component:
-          'https://www.figma.com/file/pyaYEjcwF2b5uf9y0vIfIy/SES-Dashboard?node-id=14310%3A260967&t=RQw5OvfIDN7GgH70-4',
+          'https://www.figma.com/file/pyaYEjcwF2b5uf9y0vIfIy/SES-Dashboard?node-id=14373%3A153088&t=7GuGRFEMTECsE9fy-4',
         options: {
           componentStyle: {
-            width: 343,
+            width: 375,
           },
           style: {
-            top: -18,
-            left: -22,
+            top: -36,
+            left: -40,
```

Comment:
Please, check the integration, the component don't fully match with the design

Reply:
already for the view of 375, 834, and 1994, in the other view the component don't change